src/components/FinalPoem.js

```diff
@@ -4,18 +4,31 @@ import './FinalPoem.css';
 
 const FinalPoem = (props) => {
 
-  return (
+  const onSubmission = (event) => {
+    event.preventDefault();
+    props.revealPoem();
+  }
+
+  if (props.isSubmitted) { 
+    return (
     <div className="FinalPoem">
       <section className="FinalPoem__poem">
         <h3>Final Poem</h3>
-
+          {props.submissions.map((submission, i) => (
+            <p key={i}>The {submission.adj1} {submission.noun1} {submission.adv} {submission.verb} the {submission.adj2} {submission.noun2} .</p>
```

Comment:
Just a note the test fails because the test assumes the lines of the poem are an array of strings, rather than the array of objects you are using.  

⚠️ You are also **not** using all the elements from the poem.  Not all the words are appearing.

```suggestion
            <p key={i}>The {submission.adjective1} {submission.noun1} {submission.adverb} {submission.verb} the {submission.adjective2} {submission.noun2} .</p>
```